libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -1497,11 +1497,21 @@ void AP_GPS::handle_gps_rtcm_fragment(uint8_t flags, const uint8_t *data, uint8_
 
     const uint8_t fragment = (flags >> 1U) & 0x03;
     const uint8_t sequence = (flags >> 3U) & 0x1F;
+    uint8_t* start_of_fragment_in_buffer =
```

Comment:
`const` wouldn't work because of the call to `memcpy` later. `uint8_t* const` would work if you insist (or we need to cast away the constness when calling `memcpy`).